packages/markdown/parseMarkdown.js

```diff
@@ -188,7 +188,7 @@ function getDescription(markdown) {
  * @param {string} markdown
  */
 function renderInline(markdown) {
-  return marked.parseInline(markdown);
+  return marked.parseInline(markdown, { headerIds: false, mangle: false });
```

Comment:
We could hoist `markedOptions` (without the `renderer` and use it in `renderInline` for consistency)

Reply:
I moved this `markedOptions` to the top of the file with regexp definitions